src/utils/inputAndValidate.js

```diff
@@ -0,0 +1,48 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+
+export const inputCarsNameAndValidate = async () => {
+	const userInputCarName = await MissionUtils.Console.readLineAsync('')
+
+	const carsSplit = userInputCarName.split(',')
+
+	// 입력이 빈값을 넣을때
+	carsSplit.map(car => {
+		if (car.trim().length === 0) {
+			throw new Error('[ERROR]')
+		}
+	})
+
+	// 자동차 이름 5글자 이상
+	carsSplit.map(el => {
+		if (el.length > 5) {
+			throw new Error('[ERROR]')
+		}
+	})
```

Comment:
```suggestion
	carsSplit.forEach(car => {
	      if (car.length > 5) {
			throw new Error('[ERROR]')
		}
		
		if (car.trim().length === 0) {
			throw new Error('[ERROR]')
		}
	})
```
같은 배열에 대해서 루프를 두번 돌 필요없이 한번 순회할 때 체크해줘도 괜찮을 것 같아요! 
그리고 `map`함수는 새로운 배열을 반환할 때 자주 사용되는 것 같아서 `forEach`를 사용해보는 걸 고려해보셔도 좋을 것 같아요🙃

Reply:
우혁님 말씀을 들어보니 그런거같습니다. map과 forEach 뭐를 사용해도 상관은 없지만 다른 사람들한테 이해가 편하다면 그렇게 진행하는게 좋아보이네요. 👍